sdk/eventhub/event-hubs/samples/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "eventhubs-samples",
```

Comment:
I went with `azure-{package}-samples-{ts/js}` in the file-share ones. So this one would be `azure-event-hubs-samples-ts` for the name field if we like that convention.

Reply:
Fixed.